src/qemu.rs

```diff
@@ -76,12 +76,16 @@ fn gen_sock(prefix: &str) -> PathBuf {
     path
 }
 
-fn gen_init() -> Result<NamedTempFile> {
+fn gen_init(rootfs: &Path) -> Result<(NamedTempFile, PathBuf)> {
```

Comment:
Doc comment would be good to describe the output tuple

Reply:
ha yeah, definitely.